cloudinit/net/__init__.py

```diff
@@ -336,6 +336,8 @@ def find_fallback_nic(blacklist_drivers=None):
         return find_fallback_nic_on_freebsd(blacklist_drivers)
     elif util.is_NetBSD():
         return find_fallback_nic_on_netbsd(blacklist_drivers)
+    elif util.is_OpenBSD():
+        return find_fallback_nic_on_netbsd(blacklist_drivers)
```

Comment:
Any reason not to do this?

```suggestion
    elif util.is_NetBSD() or util.is_OpenBSD():
        return find_fallback_nic_on_netbsd(blacklist_drivers)
```

And should we consider renaming `find_fallback_nic_on_netbsd` to reflect that it is no longer just used for NetBSD?

Reply:
I don't have a much better name, do you? Ultimately, I would like to move the functions in cloudinit.distros.netbsd.

Reply:
I was really just thinking something like `find_fallback_nic_on_netbsd_or_openbsd` to give future developers (who might only be considering NetBSD) a breadcrumb to realise that the function isn't NetBSD-specific.